.drone.yml

```diff
@@ -84,6 +84,25 @@ pipeline:
       local: false
       event: [ push ]
 
+  update-search-index:
+    image: algolia/docsearch-scraper
```

Comment:
Always use specific tags of used images, even if there is only `latest` available be more specific.